Dockerfile

```diff
@@ -1,31 +1,37 @@
-FROM alpine:3.19
+FROM alpine:3.21
 
 RUN apk --no-cache add \
     bash \
-    curl>7.77.0-r0 \
+    curl \
     git \
     libc6-compat \
     openssh-client \
     py3-pip \
     py3-wheel \
     python3 \
-    yamllint=1.33.0-r0
+    yamllint=1.35.1-r1
```

Comment:
I'd rather remove the specific version. There is no other version in the repo anyhow.